### PR TITLE
Update how-to-mfa-server-migration-utility.md

### PR DESCRIPTION
**Pull request by ChristianCB83**
The fact that a secondary MFA Server can be used to execute the migration utility is not mentioned in the section "Limitations and requirements". Based on this we should consider having this clear for future references as it also mentions that this is the best approach considering performance during the migration.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@Justinha 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by Justinha — APPROVED** [review]
Could anyone kindly add an end period to the last sentence?

**Comment by Justinha** [comment]
#sign-off

File: docs/identity/authentication/how-to-mfa-server-migration-utility.md [hunk 1]
```diff
@@ -33,6 +33,9 @@ Take a look at our video for an overview of the MFA Server Migration Utility and
 ## Limitations and requirements
 
 - The MFA Server Migration Utility requires a new build of the MFA Server solution to be installed on your Primary MFA Server. The build makes updates to the MFA Server data file, and includes the new MFA Server Migration Utility. You don't have to update the WebSDK or User portal. Installing the update _doesn't_ start the migration automatically.
+>[!NOTE]
+> The MFA Server Migration Utility can be executed on a secondary MFA Server. For more information, please check [Run a secondary MFA Server (optional)](#run-a-secondary-mfa-server-optional).
+
```
Comment (Jak-MS) [inline docs/identity/authentication/how-to-mfa-server-migration-utility.md:38]:
```suggestion
- The MFA Server Migration Utility requires a new build of the MFA Server solution to be installed on your Primary MFA Server. The build makes updates to the MFA Server data file, and includes the new MFA Server Migration Utility. You don't have to update the WebSDK or User portal. Installing the update _doesn't_ start the migration automatically.

   > [!NOTE]
   > The MFA Server Migration Utility can be executed on a secondary MFA Server. For more information, please check [Run a secondary MFA Server (optional)](#run-a-secondary-mfa-server-optional).

```
nesting the note under the bullet

**Pull request merged**